janus.c

```diff
@@ -3467,19 +3468,20 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
 	FILE *sgwRtspStatusFile = NULL;
     char * sgwline = NULL;
     size_t sgwlen = 0;
-    ssize_t sgwread;
-	sgwRtspStatusFile = fopen("/home/ubuntu/sgw-rtsp-status", "r");
+    ssize_t sgwStatusFileRead=0;
+	sgwRtspStatusFile = fopen(SGW_STATUS, "r");
 	if (sgwRtspStatusFile == NULL){
 			JANUS_LOG(LOG_ERR,"Failed opening sgw status file\n");
     		return FALSE;
 	}
-	while ((sgwread = getline(&sgwline, &sgwlen, sgwRtspStatusFile)) != -1) {
+	while ((sgwStatusFileRead = getline(&sgwline, &sgwlen, sgwRtspStatusFile)) != -1) {
 	    if(strstr(sgwline,"true") == NULL){
 			JANUS_LOG(LOG_ERR,"%s\n",line);
 			fclose(sgwRtspStatusFile);
 			return FALSE;
         }
     }
+	if((sgwStatusFileRead = getline(&sgwline, &sgwlen, sgwRtspStatusFile)) == -1) return FALSE;
```

Comment:
Not good.
As mentioned in my comment above - you need to check the return value of the actual (original) getline call